fdbrpc/fdbrpc.h

```diff
@@ -247,6 +247,15 @@ class RequestStream {
 		else
 			queue->send(value);
 	}
+
+	void send(T&& value) const {
```

Comment:
> So, I think that this should work out that `template<class T> T&&` is a [forwarding reference](https://en.cppreference.com/w/cpp/language/reference#Forwarding_references)

I think since T is not a template type parameter of `send`, `value` is not a forwarding reference. Here's an example from [forwarding reference](https://en.cppreference.com/w/cpp/language/reference#Forwarding_references)


```
template<class T> struct A {
    template<class U>
    A(T&& x, U&& y, int* p); // x is not a forwarding reference: T is not a
                             // type template parameter of the constructor,
                             // but y is a forwarding reference
};
```